helm/kubernetes-node-health-chart/templates/configmap.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: {{ .Values.namespace }}
   labels:
     app: kubernetes-node-health
+    giantswarm.io/service-type: "managed"
```

Comment:
Note, I added the service type label as per `fmt` guidelines for k8s resources in test clusters. 

https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md#common-labels-for-tenant-cluster-services